packages/api-gateway/src/drivers/http/adapters/users/createUser.js

```diff
@@ -0,0 +1,51 @@
+const errorHandler = require('./errorHandler');
+
+async function createUser(req, reply) {
+  try {
+    const {
+      email,
+      firstName,
+      secondName,
+      firstSurname,
+      secondSurname,
+      birthDate,
+      gender,
+      phoneNumber,
+    } = req.body;
+
+    req.log.info('[http-server]: Creating user with: ', {
+      email,
+      firstName,
+      secondName,
+      firstSurname,
+      secondSurname,
+      birthDate,
+      gender,
+      phoneNumber,
+    });
+
+    const result = await this.userServices.createUser({
+      email,
+      firstName,
+      secondName,
+      firstSurname,
+      secondSurname,
+      birthDate,
+      gender,
+      phoneNumber,
+    });
+
+    if (result.isBoom === true) {
```

Comment:
```diff
- if (result.isBoom === true)
+ if (result.isBoom)
```

Both are the same, the idea to name boolean variables using prefixes like `is, are, should` is to indicate that the response is true or false.

Reply:
commentary applied